database/migrations/2018_01_18_191700_add_columns_to_categorize_posts.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddColumnsToCategorizePosts extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('posts', function (Blueprint $table) {
+            $table->string('type')->after('northstar_id')->default('photo')->comment('Describes the type of post submitted e.g. photo, call, voter-reg');
+            $table->string('action_bucket')->after('type')->comment('Describes the bucket the action is tied to. A campaign could ask for multiple types of actions throught the life of the campaign.');
```

Comment:
I wonder if we'll want an index on either of these fields (for example, so we could query only posts for the `v1/reportbacks` or `v2/activity` endpoint, or so we could have separate galleries on PN for each action.)

Reply:
yes! indexes!